tictactoe.py

```diff
@@ -0,0 +1,202 @@
+import math
+import random
+ 
+ 
+class TicTacToe:
+    """
+    a class for the TicTacToe game
```

Comment:
Some more info about the tic tac toe game, maybe a link to the wiki page would be nice.